### ISSUE TITLE
Installation from pip is failing because of missing README.rst

### ISSUE THREAD

**Issue by Stvad** [robinhilliard/pipes#3]
Hey! Just tried to install the package from pip, and it's failing with:

```
 src/Bot-new   mainline  pip3 install pipeop
Collecting pipeop
  Downloading https://files.pythonhosted.org/packages/f3/71/5449fe7a4ca9ecef67bca9ceb13a163d7326fc0c325b6642694e7da5c0fc/pipeop-0.2.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/bw/539xc4113rd4lcc56p27lq1jghmd7n/T/pip-install-ujbf0qn9/pipeop/setup.py", line 5, in <module>
        with open('README.rst', 'r') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'README.rst'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/bw/539xc4113rd4lcc56p27lq1jghmd7n/T/pip-install-ujbf0qn9/pipeop/
```

**Comment by robinhilliard** [comment]
Tried to fix in 0.2.3 but Artifactory is caching pip and I can only download failing 0.2.2. Will try later, hopefully cache will clear.

**Comment by robinhilliard** [comment]
Ok 0.2.3 isn't working either. I don't understand why it stopped working. It works when I build the distribution because the rst contents are displayed on Pypi. Any assistance with navigating the intricacies of setup.py voodoo happily accepted.

**Comment by robinhilliard** [comment]
Created MANIFEST.in, didn't quite understand this previously. Hopefully fixed in 0.2.4

**Comment by robinhilliard** [comment]
Ok, had to go to 0.2.5 because Pypi wouldn't update 0.2.4. Tested on a new Python container and it installed without issue.
